### ISSUE TITLE
Login and character selection problem(homepage)

### ISSUE THREAD

**Issue by Reaper281** [dalaranwow/dalaran-wow#5967]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Hey! Can't login in the game and can't create a thread. I know, that i should use the community support for this problem, but i cant select any characther on the homepage, so i cant create a thread.

**Expected behaviour**: Should login and select any character on the homepage.

**Steps to reproduce the problem**:

1.
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

![IMG_20210716_124005](https://user-images.githubusercontent.com/7457833/125947432-562e0649-7751-48a2-bf52-308d6c17d4e6.jpg)

**Comment by Golganeth** [comment]
Greetings @Reaper281
In order to select your main character on the website and to post on the forums, for now you will have to create a character on Kirin Tor.
As for your account locked problem, please write us an e-mail to "dalaran.community@gmail.com" and we will help you out as soon as possible. Make sure you provide us your account name and approximate date of your last login.

Cheers!
Golgy
